docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/02 - Projects.md

```diff
@@ -0,0 +1,33 @@
+# Projects
+
+## Overview
+The **Project** represents a study in LORIS and can have different subprojects (see [Subproject setup](04 - Subprojects.md) page) which, in turn, can be affiliated with different visits (see [Timepoints setup](05 - Timepoints.md) page).
+
+**Projects** are defined in the `Project` table of the database.
+         
+## Adding Project Options
+
+### Front End (Recommended)
+Projects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab. Follow the steps below to create your own study projects.
+
+1. Enable multi-project support. In the Configuration module under the ***Study variables*** section, set the field ***Use Projects*** to true and click the ***Submit*** button at the bottom of the page.  After changing this setting, be sure to refresh the Configuration module page to see the changes.
```

Comment:
Wasn't this option deprecated/removed? Doing `git grep -i useProjects` I only see archived SQL patches that reference it.

Reply:
done